movies/src/main/java/se/mulander/cosmos/movies/impl/Movies.java

```diff
@@ -182,29 +181,26 @@ private static Movie tmdbToInternal(TMDBResponseResult tmdb, Client client, Stri
                                                     "https://image.tmdb" + "" + ".org/t/p/w1920" + tmdb.backdropPath,
                                                     Status.DEFAULT);
         m.setExtended(exMovie);
-
-        addCast(client, tmdb, exMovie, theMovieDbURL, apiKey);
-
         return m;
     }
 
     /**
      * Adds a cast to a given movie object.
      *
      * @param client        The client to create the movie API request with
-     * @param tmdb          the tmdb response for the movie object.
-     * @param exMovie       the extended movie object to further expand with cast
+     * @param movie       the extended movie object to further expand with cast
      * @param theMovieDbURL url to the movie database
      * @param apiKey        the api key for the movie database
      * @throws APIException if tmdb couldn't respond with a 200 OK
      */
-    private static void addCast(Client client, TMDBResponseResult tmdb, ExtendedMovie exMovie, String theMovieDbURL,
+    private static void addCast(Client client, Movie movie, String theMovieDbURL,
                                 String apiKey) throws APIException
 
     {
+        ExtendedMovie exMovie = movie.extendedMovie;
         Response castResponse = client.target(theMovieDbURL)
                                       .path("/3/movie")
-                                      .path("/" + tmdb.id)
+                                      .path("/" + movie.internalID)
```

Comment:
Funkar det här?
tmdb.id är ju ID:t som the movie database har medans movie.internalID är det ID jag har skapat

Reply:
ja jag blev förvånad själv men ja det funkar :)